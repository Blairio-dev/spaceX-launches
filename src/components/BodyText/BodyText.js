import React from "react";
import PropTypes from "prop-types";
import styled from "@emotion/styled";
import { colours } from "../../assets/tokens";

const StyledBody = styled("span")`
  color: ${colours.text};
  font-family: "Brandon Grotesque Regular";
  font-size: 20px;
  font-weight: 400;
`;

const BodyText = ({ text }) => <StyledBody>{text}</StyledBody>;

BodyText.propTypes = {
  /** Body text. */
  text: PropTypes.string.isRequired,
};

export { BodyText };
