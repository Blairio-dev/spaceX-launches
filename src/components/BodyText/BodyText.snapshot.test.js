import React from "react";
import renderer from "react-test-renderer";
import { BodyText } from "./BodyText";

const DEFAULT_PROPS = {
  text: "Body text",
};

describe("BodyText Snapshots", () => {
  it("Renders with default props", () => {
    const tree = renderer.create(<BodyText {...DEFAULT_PROPS} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
