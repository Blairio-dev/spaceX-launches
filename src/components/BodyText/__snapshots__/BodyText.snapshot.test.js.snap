// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BodyText Snapshots Renders with default props 1`] = `
<span
  className="css-14vudug"
>
  Body text
</span>
`;
